### PR TITLE
xonsh 0.1.5 (new formula)

### PR DESCRIPTION
**Pull request by internaught**
Had to fix a syntax error in the formula

### CODE REVIEW

File: Library/Formula/xonsh.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    resource("ply").stage do
+      system "python3", *Language::Python.setup_install_args(libexec/"vendor")
+      ENV["PYTHONPATH"] = lib/"python#{version}/site-packages"
+      ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{version}/site-packages"
+    end
+
+    resource("ply").stage do
+      system "python3", *Language::Python.setup_install_args(libexec)
+    end
```
Comment (jacknagel) [inline Library/Formula/xonsh.rb]: why is this resource being installed twice in different places?

Comment (tdsmith) [inline Library/Formula/xonsh.rb]: Yes, please remove this second stage.

File: Library/Formula/xonsh.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  def install
+    version = Language::Python.major_minor_version "python3"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{version}/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python#{version}/site-packages"
+
+    resource("ply").stage do
+      system "python3", *Language::Python.setup_install_args(libexec/"vendor")
+      ENV["PYTHONPATH"] = lib/"python#{version}/site-packages"
+      ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{version}/site-packages"
```
Comment (tdsmith) [inline Library/Formula/xonsh.rb]: You can remove lines 20 and 21; line 20 is unnecessary (and un-does the work you do above) and line 21 is redundant with line 16.

Comment (tdsmith) [inline Library/Formula/xonsh.rb]: correction: line 21 is redundant with line 15.

File: Library/Formula/xonsh.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{version}/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python#{version}/site-packages"
+
+    resource("ply").stage do
+      system "python3", *Language::Python.setup_install_args(libexec/"vendor")
+    end
+
+    system "python3", *Language::Python.setup_install_args(libexec)
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(prefix, :PYTHONPATH => ENV["PYTHONPATH"])
```
Comment (tdsmith) [inline Library/Formula/xonsh.rb]: Use `libexec/"bin"` here, not `prefix`

**Comment by tdsmith** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**